.github/dependabot.yml

```
@@ -0,0 +1,11 @@
+version: 2
+updates:
```

@Ferroin can we ignore this error?

---

Theoretically yes, but it’s easier to just fix it...